aws/cluster/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "default_ingress_nameservers" {
+  value = module.cluster.default_ingress_nameservers
```

Comment:
The property `module.cluster.default_ingress_nameservers` doesn't exist in the repository. Shouldn't the output value rather be exposed through the internal `_module` to the external facing cluster output all the way to the end user?